package.json

```diff
@@ -26,7 +27,6 @@
     "reading-time-estimator": "^1.11.0",
     "recma-mdx-change-props": "^1.1.1",
     "recma-mdx-escape-missing-components": "^1.1.0",
-    "rehype-highlight": "^7.0.1",
```

Comment:
This demo application is just for demonstrating `next-mdx-remote-client` features. I do not plan to change code highlighting plugin. There is one more reason that I am also the author of  `rehype-highlight-code-lines` and wanna see it is working with `rehype-highlight`.